app/styleguide/page.tsx

```diff
@@ -520,14 +994,17 @@ function Page() {
                 find specific
                 properties, and easier to add &amp; remove. Eg.
 
-                <div><pre><code>
+                <div>
+                  <pre>
+                  <code>
                   {`<SomeElement
```

Comment:
@fraserclark Did you check if this is not broken now? I remember consciously formatting this without the indentation because it broke the layout in the final rendered HTML.